app/uk/gov/hmrc/tariffclassificationfrontend/service/CasesService.scala

```diff
@@ -62,10 +62,25 @@ class CasesService @Inject()(appConfig: AppConfig, auditService: AuditService, e
     val caseUpdating = original.copy(status = CaseStatus.COMPLETED, decision = Some(decisionUpdating))
 
     for {
-      updated <- connector.updateCase(caseUpdating)
-      _ <- emailService.sendCaseCompleteEmail(updated)
-        .recover({case t: Throwable => Logger.error(s"Could not send Complete Case Email for case [${updated.reference}]", t)})
-      _ <- addEvent(original, updated, operator, Some("The applicant was sent an Email Confirmation with their reference"))
+      // Update the case
+      updated: Case <- connector.updateCase(caseUpdating)
+
+      // Send the email
+      message <- emailService.sendCaseCompleteEmail(updated)
```

Comment:
This code is unreadable.

Reply:
Generally its more constructive to make a suggestion on how to improve ;)